### PR TITLE
Contact us page

### LINKED ISSUE #531: Add Contact us button to the website menu
The link to the contact form is difficult to find (e.g. at the end of the FAQ). Adding a `Contact us` button to the website menu will greatly improve the user experience. This page could also have a link to the team X accounts (@InterProDB and @PfamDB) and to the linkedIn account (https://www.linkedin.com/company/interpro-pfam/)

### PR DESCRIPTION
**Pull request by gustavo-salazar**
A page with all the contact details.

FIX #531

### CODE REVIEW

File: src/pages/Contact/index.tsx [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+            href="//www.ebi.ac.uk/support/interpro"
+            target="_blank"
+            withReferrer
+          >
+            <span className={css('icon', 'icon-common')} data-icon="&#x6e;" />{' '}
+            Submit a ticket
+          </Link>
+        </li>
+        <li>
+          <Link href="mailto: interhelp@ebi.ac.uk">
```
Comment (typhainepl) [inline src/pages/Contact/index.tsx:36]: I wouldn't add the helpdesk email address, as this is likely to lead to receive more spams from robots

File: src/pages/Contact/index.tsx [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+            href="//www.ebi.ac.uk/support/pfam"
+            target="_blank"
+            withReferrer
+          >
+            <span className={css('icon', 'icon-common')} data-icon="&#x6e;" />{' '}
+            Submit a ticket
+          </Link>
+        </li>
+        <li>
+          <Link href="mailto: pfam-help@ebi.ac.uk">
```
Comment (typhainepl) [inline src/pages/Contact/index.tsx:69]: see previous comment

File: src/pages/Contact/index.tsx [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+          </Link>
+        </li>
+        <li>
+          <Link href="mailto: interhelp@ebi.ac.uk">
+            <span className={css('icon', 'icon-common')} data-icon="&#xf0e0;" />{' '}
+            interhelp@ebi.ac.uk
+          </Link>
+        </li>
+        <li>
+          <Twitter />
```
Comment (typhainepl) [inline src/pages/Contact/index.tsx:42]: Twitter is now called X and the term "tweet" as been changed to "post"

File: src/pages/Contact/index.tsx [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+          </Link>
+        </li>
+        <li>
+          <Link href="mailto: pfam-help@ebi.ac.uk">
+            <span className={css('icon', 'icon-common')} data-icon="&#xf0e0;" />{' '}
+            pfam-help@ebi.ac.uk
+          </Link>
+        </li>
+        <li>
+          <Twitter handler="PfamDB" />
```
Comment (typhainepl) [inline src/pages/Contact/index.tsx:75]: Twitter is now called X and the term "tweet" as been changed to "post"

File: src/components/Twitter/index.tsx [hunk 5]
```diff
@@ -17,11 +21,11 @@ const Twitter = () => {
         data-chrome={'nofooter noborders noheader noscrollbar transparent'}
         data-tweet-limit="1"
         className={css('twitter-timeline')}
-        href="https://twitter.com/InterProDB"
+        href={`https://twitter.com/${handler}`}
         target="_blank"
       >
         {' '}
-        Tweets by @InterProDB
+        Tweets by @{handler}
```
Comment (typhainepl) [inline src/components/Twitter/index.tsx:28]: Change to "Posts by"

File: src/components/Twitter/__snapshots__/test.js.snap [hunk 6]
```diff
@@ -17,7 +17,8 @@ exports[`<Twitter /> should render 1`] = `
     target="_blank"
   >
      
-    Tweets by @InterProDB
+    Tweets by @
```
Comment (typhainepl) [inline src/components/Twitter/__snapshots__/test.js.snap:20]: Change to "Posts by"

**Review by typhainepl — CHANGES_REQUESTED** [review]
A page that was definitely missing!
I added a few comments, notably Twitter is now called "X" and I believe we should change the logo.

**Comment by matthiasblum** [comment]
Is an extra item in the menu and an extra page _really_ what we need?

We already have the _Help_ and _About_ buttons, each with their own submenu. That's twelve pages. Some contain links to ReadTheDocs. Some actually _import_ their content from ReadTheDocs.

We have a link to the support & feedback form:

* in the [FAQ](https://www.ebi.ac.uk/interpro/help/faqs/) (imported from ReadTheDocs)
* on the [Training page](https://www.ebi.ac.uk/interpro/help/training/) (imported from ReadTheDocs)
* on the [Documentation page](https://www.ebi.ac.uk/interpro/help/documentation/)
* and now on the new Contact Us page

Also, the Help/About section are kind of overlapping/interconnecting (the content between `Help >FAQ` and `About > InterPro` is quite redundant).

Maybe we could simplify this part of the website, e.g. one "Help center" page, that contains:

- links to training material (tutorial, webinars, links to future courses)
- links to various documentations (InterPro, Pfam, InterProScan)
- links to InterPro/Pfam support & feedback forms, and social medias

**Comment by gustavo-salazar** [comment]
I was thinking about @matthiasblum comment, and given that we forgot to talk about it in standup I'll write it here.

I think that a bit of redundancy on training and help materials is not a bad thing, as long as the users manage to find the info they need. But also think that restructuring into a `Help Centre` or something like it is not a bad idea as it might again help user to find the help they need.

The suggested page is a quick solution to the fact that the contact information is quite hidden, even if is replicated in multiple places.

So, I suggest going ahead with the content of this PR, and implementing @typhainepl suggestions. But also create a story to plan restructuring that area, maybe following @matthiasblum suggestion in this PR, but also giving a bit more thought to it and see if that suffices the needs of help & documentation.

What do you think?

**Comment by matthiasblum** [comment]
> 
> So, I suggest going ahead with the content of this PR, and implementing @typhainepl suggestions. But also create a story to plan restructuring that area, maybe following @matthiasblum suggestion in this PR, but also giving a bit more thought to it and see if that suffices the needs of help & documentation.
> 
> What do you think?

Yep, sounds good. :+1:

**Comment by typhainepl** [comment]
It looks good to me.
I guess we could remove the `Contact us` link from the `Documentation` and `FAQ` pages.

**Comment by gustavo-salazar** [comment]
> It looks good to me. I guess we could remove the `Contact us` link from the `Documentation` and `FAQ` pages.

I'm going to leave them there. I still think redundancy is a good thing in this case. But definitely something to consider while planning for the "Help center" idea.

BTW, I have opened issue #563 for the Help center idea.

**Pull request merged**